### ISSUE TITLE
build fails ' Cannot find file: with-alerts".'

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#370]
```
What steps will reproduce the problem?
1. build with build.sh script on Debian Wheezy 64
2.
3.

What is the expected output? What do you see instead?

Cannot find file: with-alerts".
# qmake freemedforms.pro -r CONFIG+=debug CONFIG-=release 
CONFIG+=debug_without_install CONFIG+="with-webcam with-alerts" 
*** ERROR: qmake.
* Build UNCOMPLETE: freemedforms


Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `contact@medecinelibre.com` on 29 Jul 2014 at 7:45

**Comment by GoogleCodeExporter** [comment]
```
After doing this:

    If you have any strange behavior during the build process, try to start the build from scratch:
        remove all Makefile: find . -type f -name Makefile -delete
        remove all qmake cache: find . -type f -name .qmake.cache -delete
        remove the build dir: rm -rf ./build
        remove the bin output dir: rm -rf ./bin

I'm able to start the build by issuing this:
./build.sh -tw -b freemedforms

Would it be possible to add the delete in the build.sh script? Maybe as an 
optional feature? Or should we make it the default it?
```

Original comment by `contact@medecinelibre.com` on 29 Jul 2014 at 11:06

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 7e9d417a388e.


updating build script:
- better clean process
- centralizing command execution
- log Qt information
- FIXME & NOTE
```

Original comment by `eric.mae...@gmail.com` on 30 Jul 2014 at 6:04

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 1a69cc25a792.


Debug dry run using command line
```

Original comment by `eric.mae...@gmail.com` on 30 Jul 2014 at 8:23

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 0f85caf0f1fa.
```

Original comment by `eric.mae...@gmail.com` on 30 Jul 2014 at 8:23
- Changed state: **Fixed**
